deep_quoridor/src/agents/dexp.py

```diff
@@ -56,8 +55,13 @@ class DExpAgentParams(TrainableAgentParams):
 
     # Whether oppoments actions are used for training
     # This is used for training only, not for playing
+    # This should not be combined with assign_negative_rewards
```

Comment:
If it's easy, can you put an assert somewhere (E.g. in the constructor of DexpAgent) to make sure it doesn't happen?